frontend/src/metabase/containers/NewItemMenu/NewItemMenu.unit.spec.tsx

```diff
@@ -78,16 +89,35 @@ async function setup({
     },
   );
 
+  if (isEmbeddingIframe) {
+    jest.spyOn(domUtils, "isWithinIframe").mockReturnValue(true);
+  }
+
   renderWithProviders(
     <>
       <NewItemMenu trigger={<button>New</button>} />
       <NewModals />
     </>,
+    entityTypes
+      ? {
+          storeInitialState: createMockState({
+            embed: createMockEmbedState({
+              options: createMockEmbedOptions({
+                entity_types: entityTypes,
+              }),
+            }),
+          }),
+        }
+      : undefined,
   );
   await userEvent.click(screen.getByText("New"));
 }
 
 describe("NewItemMenu", () => {
+  afterEach(() => {
+    jest.restoreAllMocks();
```

Comment:
we need to [reset mocks](https://jestjs.io/docs/jest-object#jestspyonobject-methodname) after each test otherwise, modules mocked by `jest.spyOn` will stay.

<img width="842" alt="image" src="https://github.com/user-attachments/assets/25ce4e7c-6897-4dac-9047-57f8102b2650" />